querybook/webapp/ui/FormikField/DatePickerField.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import moment from 'moment';
+import { useField } from 'formik';
+import styled from 'styled-components';
+
+import { SearchDatePicker } from 'components/Search/SearchDatePicker';
+
+interface IDatePickerFieldProps {
+    name: string;
+}
+
+const DatePickerWrapper = styled.div`
+    display: flex;
+    align-items: center;
+    width: 50%;
+`;
+
+const StyledDatePicker = styled(SearchDatePicker)`
+    margin-bottom: 0px;
+`;
+
+function secondsToDate(dateInSeconds?: number): number {
```

Comment:
Date is still seconds? so I suppose this function removes utcOffset?

Reply:
Yes, date is still in seconds. This function should restore date from UTC. I save the date in UTC but for user I want to show the time in UTC.

Reply:
rename it to utcSecondsToLocal?